src/lsptoolshost/dotnetRuntimeExtensionResolver.ts

```diff
@@ -39,38 +36,47 @@ export class DotnetRuntimeExtensionResolver implements IHostExecutableResolver {
     private hostInfo: HostExecutableInformation | undefined;
 
     async getHostExecutableInfo(): Promise<HostExecutableInformation> {
-        let dotnetRuntimePath = commonOptions.dotnetPath;
```

Comment:
A breaking change document for dotnetPath should be created, IMO. I would also make sure this is remarked in the release notes, and that our existingPath setting is mentioned.
 https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime&ssr=false#overview:~:text=I%20already%20have%20a%20.NET%20Runtime%20or%20SDK%20installed%2C%20and%20I%20want%20to%20use%20it

Reply:
I haven't removed this in this PR - plan to in a followup.  What might be an even better option is to migrate existing users of `dotnet.dotnetPath` to the new setting.  We already do that for migrating O# options